birdhouse/env.local.example

```diff
@@ -533,7 +533,7 @@ export THREDDS_ADDITIONAL_CATALOG=""
 
 # Raven to use the local Geoserver instead of the default production.
 # See raven/default.env for more info.
-#export RAVEN_GEO_URL="https://${PAVICS_FQDN}/geoserver/"
+#export RAVEN_GEO_URL="https://${BIRDHOUSE_FQDN}/geoserver/"
```

Comment:
Note to self: make an issue for this (same as: https://github.com/bird-house/birdhouse-deploy/pull/428#discussion_r1513204277)

Reply:
This one can actually be the internal `BIRDHOUSE_FQDN` and not the public one because both RavenWPS and Geoserver are on the same host, for now.

The day they are on different host, then we will need the public one.  I guess we can go with the public one now.  Probably negligible performance hit.